### ISSUE TITLE
typo in docs/core/howto/defer.rst leads to bad rendering

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8723]
|[<img alt="evgeni's avatar" src="https://avatars.githubusercontent.com/u/94284?s=50" width="50" height="50">](https://github.com/evgeni)| @evgeni reported|
|-|-|
|Trac ID|trac#8723|
|Type|defect|
|Created|2016-08-04 10:09:27Z|

ohai

https://twistedmatrix.com/documents/current/core/howto/defer.html has ```"known as a series of errbacks or an**errback chain**"```, the missing space between ```"an"``` and ```"**"``` leads to bad rendering of the rst

<details><summary>Searchable metadata</summary>

```
trac-id__8723 8723
type__defect defect
reporter__evgeni evgeni
priority__lowest lowest
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1470305367600028 1470305367600028
changetime__1474198569965514 1474198569965514
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="evgeni's avatar" src="https://avatars.githubusercontent.com/u/94284?s=50" width="50" height="50">](https://github.com/evgeni)<a name="note_1"></a>|@evgeni set owner to @evgeni<br>@evgeni set status to `assigned`|
|-|-|

https://github.com/twisted/twisted/pull/460

**Comment by twisted-trac** [comment]
|[<img alt="evgeni's avatar" src="https://avatars.githubusercontent.com/u/94284?s=50" width="50" height="50">](https://github.com/evgeni)|@evgeni removed owner<br>@evgeni set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset 89c261ee5ae4c8dbc8457a53fc3f9fdd8cc26f18
```
#!CommitTicketReference repository="" revision="89c261ee5ae4c8dbc8457a53fc3f9fdd8cc26f18"
Merge evgeni/8723-evgeni-typo: Fix bad rst formatting in defer.rst


Author: evgeni
Reviewer: adiroiban
Fixes: #8723
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing the review tag from closed tickets
